docs/Datadog.Trace/README.md

```diff
@@ -0,0 +1,227 @@
+# Datadog.Trace NuGet package
+
+This package contains the Datadog .NET APM tracer for configuring custom instrumentation.
+
+> If you are only using automatic tracing, **you do not need this package**. Please [read our documentation](https://docs.datadoghq.com/tracing/setup/dotnet) for details on how to install the tracer for automatic instrumentation. 
+
+## Getting Started
+
+1. Configure the Datadog agent for APM [as described in our documentation](https://docs.datadoghq.com/tracing/setup_overview/setup/dotnet-core#configure-the-datadog-agent-for-apm).
+2. For automatic instrumentation, install and enable the tracer [as described in our documentation](https://docs.datadoghq.com/tracing/setup_overview/setup/dotnet-core/?tab=windows#install-the-tracer). 
+3. Configure custom instrumentation, as shown below
+4. [View your live data on Datadog](https://app.datadoghq.com/apm/traces).
+
+### Configuring Datadog in code
+
+There are multiple ways to configure your application, for example using Environment variables, _web.config_, or _datadog.json_, [as described in our documentation](https://docs.datadoghq.com/tracing/setup_overview/setup/dotnet-core/#configuration). This NuGet package also allows you to configure settings in code.
+
+To override configuration settings, create an instance of `TracerSettings`, and pass it to the static `Tracer.Configure()` method:
+
+```csharp
+using Datadog.Trace;
+
+// Create a settings object using the existing 
+// environment variables and config sources
+var settings = TracerSettings.FromDefaultSources();
+
+// Override a value
+settings.GlobalTags.Add("SomeKey", "SomeValue");
+
+// Replace the tracer configuration
+Tracer.Configure(settings);
+```
+
+Calling `Tracer.Configure()` will replace the settings for all subsequent traces, both for custom instrumentation and for automatic instrumentation.
+
+> :warning: Replacing the configuration should be done once, as early as possible in your application. 
+
+If you used to set the settings programatically, also note that transport related settings have been isolated in the `ExporterSettings`. It is still encapsulated in `TracerSettings`.
```

Comment:
I think this could be included in the `Upgrading from 1.x to 2.0` section. It seem unnecessary to include right here

Reply:
Fixed in 6d95cccfa92dc84cfcb1571a8608462735ff452a